deploy-manage/monitor/autoops/cc-cloud-connect-autoops-troubleshooting.md

```diff
@@ -0,0 +1,56 @@
+---
+applies_to:
+  deployment:
+    self:
+navigation_title: Troubleshooting guide
+---
+
+# AutoOps for self-managed clusters troubleshooting guide
+
+## Troubleshoot issues
```

Comment:
shouldn't have two headings beside each other. need a sentence above the heading